stl/CMakeLists.txt

```diff
@@ -556,25 +603,27 @@ function(add_stl_dlls D_SUFFIX THIS_CONFIG_DEFINITIONS THIS_CONFIG_COMPILE_OPTIO
     set_target_properties(msvcp${D_SUFFIX}_implib PROPERTIES ARCHIVE_OUTPUT_NAME "msvcprt${D_SUFFIX}")
 endfunction()
 
-add_stl_dlls("" "" "${VCLIBS_RELEASE_OPTIONS}" "/GL" "/LTCG;/opt:ref,icf")
-add_stl_dlls("d" "_DEBUG" "${VCLIBS_DEBUG_OPTIONS}" "" "/opt:ref,noicf")
+add_stl_dlls("" Release "/GL" "/LTCG;/opt:ref,icf")
+add_stl_dlls("d" Debug "${VCLIBS_DEBUG_OPTIONS}" "" "/opt:ref,noicf")

```

Comment:
This was declared as:

`function(add_stl_dlls D_SUFFIX REL_OR_DBG GL_FLAG THIS_CONFIG_LINK_OPTIONS)`

Thus I am confused as to why `"${VCLIBS_DEBUG_OPTIONS}"` is being retained here.